controllers/counsels.controller.js

```diff
@@ -24,6 +24,19 @@ exports.createCounsel = async (req, res, next) => {
   }
 };
 
+exports.getCounselList = async (req, res, next) => {
+  try {
+    const story = await Counsel.find().populate("counselee").lean();
+
+    res.status(201).json({
+      result: "success",
+      story: story,
+    });
```

Comment:
```suggestion
    const stories = await Counsel.find().populate("counselee").lean();

    res.status(200).json({
      result: "success",
      data: stories,
    });
```

리스트를 전달하니 복수형태로 사용하고 GET요청 라우터 관련으로 200번으로 사용하면 좋을 것 같습니다!